_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,134 @@
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-6.png" width=500 alt="Index pattern search user interface">
+
+4. Add an **Index pattern name** to define the index pattern and then select **Next step**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-7.png" width=500 alt="Index pattern define user interface">
+
+5. Select an option for the **Time field**, then choose **Create index pattern**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-8.png" width=500 alt="Index pattern time field user interface">
+
+## Search data
+
+Before you start searching for data, set up the time filter. The sample index pattern used for this tutorial contains time-based data. You can set a time filter that displays only the data within a specified time range and you can select the time filter to change the time range or select a specific time range in the histogram.
+
+### Use the time filter
+
+1. In the Dashboards console, select **Discover** and confirm the index pattern being used is `opensearch_dashboards_sample_data_ecommerce`.
+2. Select the calendar icon to change the time field. The default is **Last 15 minutes**.
+3. Change the time field to **Last 7 days** and select **Refresh**.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-9.png" alt="Time filter user interface">
+
+4. To set the start and end times, select the bar next to the time filter. In the popup, select **Absolute**, **Relative**, or **Now**, then specify the required options.
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-10.png" width=300 alt="Start and end times user interface">
+
+### Select a time range from the histogram
+
+To select a time range for the histogram, you can do one of the following:
+
+* Select the bar that represents the time range you want to zoom in on.
+* Select the bar and drag to view a specific time range. You must start the selection with the cursor over the background of the chart—the cursor changes to a plus sign when you hover over a valid start point.
+* Select the dropdown, then select an interval.
+
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-11.jpg" alt="Histogram user interface">
+
+## Create visualizations
+
+Follow these steps to learn how to connect your visualizations in a single dashboard.
+
+1. In the Dashboards console, select **Visualize** > **Create visualization**.
+2. Select the visualization type. For this tutorial, select **Line**.
+3. Choose a source. For this tutorial, select the index pattern `opensearch_dashboards_sample_data_ecommerce`.
+4. Under **Buckets**, select **Add > X-axis**.
+5. In the **Aggregation** field, select **Date Histogram** and then **Update**.
+6. Select **Save** and add file name. This tutorial uses preconfigured visualizations, so you won’t be able to save your visualization.
+
+## Connect visualizations in a single dashboard
```

Comment:
Can we include a brief sentence between the heading and the list that introduces the list?